aspnetcore/fundamentals/hosting.md

```diff
@@ -15,7 +15,7 @@ uid: fundamentals/hosting
 
 By [Luke Latham](https://github.com/guardrex)
 
-ASP.NET Core apps configure and launch a *host*, which is responsible for app startup and lifetime management. At a minimum, the host configures a server and a request processing pipeline.
+ASP.NET Core apps configure and launch a *host*. The *host* is responsible for app startup and lifetime management. At a minimum, the host configures a server and a request processing pipeline.
```

Comment:
The emphasis on "host" can be removed after the first use, so I recommend just "The host is ..."